mailgun/tests.py

```diff
@@ -315,17 +451,17 @@ def test_multi_mailing_list_recipients(self, mock_ml_get, mock_ci_get, mock_ss_f
         response = handle_mailing_list_email_route(request)
         self.assertEqual(response.status_code, 200)
         send_mail_call = call(
-            'Unit Test via Canvas',
-            'unittest@example.edu',
-            ['teacher@example.edu', 'student@example.edu', 'unittest@example.edu'],
-            '[Lorem For Beginners] blah',
+            u'Unit Test via Canvas',
```

Comment:
the tests were failing and the only diff I could see was the actual call and expected calls were diff in that one had u'' and one did not. It turns out there was another diff, the order of the lists were diff, but I didn't see that at the time. The tests pass now, so I didn't bother to change the strings back.